sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/action/ReflectiveActionRouter.scala

```diff
@@ -62,15 +70,17 @@ class ReflectiveActionRouter[A <: Action](action: A, commandHandlers: Map[String
         componentMethod.requestMessageDescriptor,
         message.metadata())
 
-    val typeUrl = message.payload().asInstanceOf[ScalaPbAny].typeUrl
-
-    val response =
-      componentMethod
-        .lookupInvoker(typeUrl)
-        .invoke(action, context)
-        .asInstanceOf[Flux[Action.Effect[_]]]
+    val inputTypeUrl = message.payload().asInstanceOf[ScalaPbAny].typeUrl
 
-    Source.fromPublisher(response)
+    componentMethod.lookupInvoker(inputTypeUrl) match {
+      case Some(methodInvoker) =>
+        val response = methodInvoker.invoke(action, context).asInstanceOf[Flux[Action.Effect[_]]]
+        Source.fromPublisher(response)
+      case None if ignoreUnknown => Source.empty()
```

Comment:
We need to check if that is legal or expected. 

I can imagine a subscription / publishing that for every incoming message produces one or more outgoing messages.

Reply:
Sorry. This is what I reviewed with Johan.

Reply:
I don't remember that we talked about that. I think it is alway ok for a streaming out to complete with empty stream but best verify by actually trying that with the proxy (I can't find any test coverage of subscription with streaming in)

I could imagine some cool stuff you could do if streaming in is allowed though, `statefulMap` + `flatMapConcat` to aggregate incoming events and only publish outgoing based on the aggregated state type of stuff.

Reply:
That doesn't mean it should/must be supported in the spring sdk though

Reply:
I've got confused then. I thought that we covered this case when talking about `ResultPart` and `ResultEnd` in the `EventingManager`. My bad, we only covered handleUnary then I suppose. In either case, @octonato and I decided to continue this in another PR. In part related to this https://github.com/lightbend/kalix-jvm-sdk/issues/1169
Are you fine with this @johanandren?